charts/firehose/templ.yaml

```
@@ -0,0 +1,1125 @@
+        runAsGroup: 101337
+        runAsNonRoot: true
+        runAsUser: 101337
+      terminationGracePeriodSeconds: 60
+      volumes:
+        - name: tmp # this is to provide a writable /tmp even with securityContext.readOnlyRootFilesystem=true
+          emptyDir: {}
+        - name: jwt
+          secret:
+            secretName: firehose-reader-jwt
+            items:
+              - key: jwt.hex
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 14

(indentation)